src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -221,6 +221,7 @@ def end_value(device)
       end
 
       DEVICE_ICONS = {
+        bcache:    Icons::HD, # TODO: have own icon for bcache
```

Comment:
Please define `Icons::BCACHE`. The fact that we are reusing the HD png file so far should not be visible here. That's what we `y2partitioner/icons.rb` for.